seed/ts-sdk/auth-environment-variables/package.json

```diff
@@ -31,9 +31,10 @@
         "build": "yarn build:cjs && yarn build:esm",
         "build:cjs": "tsc --project ./tsconfig.cjs.json",
         "build:esm": "tsc --project ./tsconfig.esm.json && node scripts/rename-to-esm-files.js dist/esm",
-        "test": "jest tests/unit",
-        "test:browser": "jest --config jest.browser.config.mjs",
-        "test:wire": "jest tests/wire",
+        "test": "jest --config jest.config.mjs",
+        "test:unit": "jest --selectProjects unit",
+        "test:browser": "jest --selectProjects browser",
+        "test:wire": "jest --selectProjects wire",
```

Comment:
@Swimburger can we get rid of the `wire:test` command, any reason we have that?

Reply:
Removed